src/Data/Product/Properties.agda

```diff
@@ -18,3 +18,11 @@ module _ {a b} {A : Set a} {B : A → Set b} where
  ,-injectiveʳ : ∀ {a} {b c : B a} → (Σ A B ∋ (a , b)) ≡ (a , c) → b ≡ c
  ,-injectiveʳ refl = refl
 
+ proj₁-injective : ∀ {a} {b : B a} {c : Σ A B} → c ≡ (a , b) → proj₁ c ≡ a
+ proj₁-injective refl = refl
+
+ proj₂-injective : ∀ {a} {b : B a} {c} → c ≡ (a , b) → proj₂ c ≡ b
+ proj₂-injective refl = refl
+
+ proj-injective : ∀ (ab : Σ A B) → ab ≡ (proj₁ ab , proj₂ ab)
+ proj-injective (_ , _) = refl
```

Comment:
Isn't this true by eta-equality (i.e. no need to pattern-match)?

Reply:
Yup, you're right. Removed.